thinkhazard_processing/settings.py

```diff
@@ -0,0 +1,9 @@
+
+settings = {
+    'data_path': '/tmp',
+    'sqlalchemy.url': ('postgresql://'
+                       'www-data:www-data@localhost:5432/'
+                       'thinkhazard_processing'),
+    'threshold': 98.0665,
+    'return_periods': [2475, 475, 100]
+}
```

Comment:
I'm not in favor of this `settings.py` file.
Instead I think that we should use `.ini` files to store the settings like we do in the thinkhazard project.